staff/jordi-pujol/learnyounode/http-json-api-server/index.js

```diff
@@ -0,0 +1,34 @@
+const http = require('http')
+
+const { argv: [, , port] } = process
+
+http.createServer((req, res) => {
+    if (req.url.startsWith('/api/')) {
+        const [path, query] = req.url.split('?')
+        const [, timestamp] = query.split('=')
+
+        if (path === '/api/unixtime') {
+            const response = {
+                unixtime: new Date(timestamp).getTime()
+            }
+
+            res.writeHead(200, { 'Content-Type': 'application/json' })
+
+            res.end(JSON.stringify(response))
+        } else {
```

Comment:
Always that the path is not 'api/unixtime' you want to do that or only when is 'api/parsetime'?